.github/workflows/pack-debian.yml

```diff
@@ -108,22 +108,16 @@ jobs:
           dpkg -i ./artifacts/khiops-core* || true
           apt-get -f install -y
       - name: Check Khiops core installation
-        run: |
-          khiops-env --env
-          khiops -s
-          khiops_coclustering -s
+        uses: ./.github/actions/test-khiops-install
       - name: Install Khiops Desktop (with java)
         run: |
           dpkg -i ./artifacts//khiops_* || true
           apt-get -f install -y
       - name: Check Khiops installation
-        run: |
-          khiops-env --env
-          khiops -s
-          khiops_coclustering -s
-      - name: Test Khiops installation
-        continue-on-error: ${{ matrix.os == 'debian:11' }}
         uses: ./.github/actions/test-khiops-install
+      - name: Test Khiops on Iris dataset
+        continue-on-error: ${{ matrix.os == 'debian:11' }}
```

Comment:
Why do we need this condition?

Reply:
I think there was a problem with debian 11 on  last release. I remove this

Reply:
done